db/models/order.js

```diff
@@ -0,0 +1,16 @@
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Order = db.define('orders', {
+  orderDate: {
+    // is this necessary if we have a created_at we can use??
+    type: Sequelize.DATE,
+  },
+  status:{
+    // only admin should be able to change the status
+    type: Sequelize.ENUM,
+    values: ['processing', 'shipped', 'delivered']
```

Comment:
@cleverettgirl missing virtual getter method based on all orderlines' subtotals. Are you adding this after orderlines are done?